ui/app/App.js

```diff
@@ -497,6 +497,18 @@ class App extends React.Component {
     };
 
     this.Notebook.run = async (path, options) => {
+      if (this.Notebook.runDebounce) {
```

Comment:
What are we debouncing? I think of that as being a hardware thing when a button is noisy

Reply:
Same problem, I think -- something is triggering two simultaneous runs, but I haven't managed to track it down yet.